components/Media/samples/BackdropSaturationBrush.md

```diff
@@ -0,0 +1,15 @@
+---
+title: BackdropSaturationBrush
+author: erinwoo  
+description: A brush that can increase or decrease the saturation of the brush's background in the application.
+keywords: back, backdrop, saturation, brush
+dev_langs:
+  - csharp
+category: Media
+subcategory: Brushes
+discussion-id: 0
+issue-id: 0
+icon: Assets/EffectAnimations.png
+---
+The `Saturation` property specifies a double value for the amount of Saturation to apply from 0.0 - 1.0. Zero being monochrome, and one being fully saturated. The default is 0.5.
+> [!SAMPLE BackdropSaturationBrushSample]
```

Comment:
Not sure if spacing between paragraphs/samples is needed here, would help a bit with raw readability. I know sometimes it does trip up some parsers otherwise.